addon/components/sl-button.js

```diff
@@ -129,13 +140,13 @@ export default Ember.Component.extend( TooltipEnabled, {
     /**
      * The current label text for the button
      *
-     * @function currentLabel
+     * @function
      * @observes label, pending, pendingLabel
```

Comment:
@observes can be removed due to the intended use of https://github.com/notmessenger/jsdoc-plugins#emberobservestag